unittest/python/crocoddyl_legacy/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ SET(${PROJECT_NAME}_PYTHON_FILES
   )
 
 FOREACH(python ${${PROJECT_NAME}_PYTHON_FILES})
-  PYTHON_BUILD(../crocoddyl ${python})
+  PYTHON_BUILD(../crocoddyl_legacy ${python})
```

Comment:
This line is making possible to build our Python bindings, it doesn't have anything to do with legacy code.
Furthermore, we don't want to build the legacy code, they will be erase asap.

In short, you need to revert this change together with the first comment.

Reply:
done